test/e2e/operator/nmstate_install_test.go

```diff
@@ -85,24 +74,55 @@ var _ = Describe("NMState operator", func() {
 			})
 		})
 	})
+	Context("when another handler is installed with different namespace", func() {
+		var (
+			operatorNamespace = "nmstate-alt"
+		)
+		BeforeEach(func() {
+			installNMState(defaultNMState)
+			daemonset.GetEventually(handlerKey).Should(daemonset.BeReady())
+			installOperator(operatorNamespace)
+		})
+		AfterEach(func() {
+			uninstallNMState(defaultNMState)
+			uninstallOperator(operatorNamespace)
+			installOperator("nmstate")
```

Comment:
Why is the operator installed in "nmstate" namespace at the end?
If it's done to test that operator can be installed and uninstalled at one namespace and then installed in another, wouldn't it be better to have another spec for it?

Reply:
The test-e2e-operator test suite spec the operator to be installed, but the test we are adding here remove the nmstate operator to check that the handler installed from different namespace is blocked until the "nmstate" one is delete. 

The install here ensure that after that test the nmstate operator is there as expected at the other test cases.